agents/process/process_logger.go

```diff
@@ -88,6 +92,13 @@ func (pl *processLogger) Latest() []string {
 	return result
 }
 
+func (pl *processLogger) clearKeywords(s string) string {
+	for _, k := range pl.hideKeywords {
+		s = strings.Replace(s, k, "***", -1)
```

Comment:
> Replace returns a copy of the string s with the first n non-overlapping instances of old replaced by new. **If old is empty, it matches at the beginning of the string and after each UTF-8 sequence, yielding up to k+1 replacements for a k-rune string.**

That's very counter-intuitive when you encounter it. Let's skip, or log and skip, or even panic if `k == ""`.

Reply:
ok